src/cosmic_ray/cli.py

```diff
@@ -185,10 +185,18 @@ def handle_baseline(args):
                 end_pos=template.end_pos,
                 job_id=template.job_id))
 
-    # Run the single-entry session.
-    cosmic_ray.commands.execute(baseline_session_file)
-
-    return ExitCode.OK
+        # Run the single-entry session.
+        cosmic_ray.commands.execute(db)
+
+        result: WorkResult = next(db.results)[1]
+        if result.test_outcome == TestOutcome.KILLED:
+            print("Execution with no mutation gives those following errors:")
+            for line in result.output.split('\n'):
+                print("  >>>", line)
+            return 1
+        else:
+            print("Execution with no mutation works fine:")
```

Comment:
I try to follow the policy of not producing any output on success, so let's remove this print statement. (The printing on error is fine.)

Reply:
print under --report